src/sdc11073/mdib/consumermdib.py

```diff
@@ -384,6 +384,11 @@ def process_incoming_states_report(self, report_type: str,
                     if self._has_new_state_usable_state_version(old_state_container, state_container,
                                                                 report_type,
                                                                 is_buffered_report):
+                        if state_container.is_context_state:
```

Comment:
Add the log message to _update_from_context_states_report, it fits better there.

Reply:
refactoring of this method and info log message will be done in https://github.com/Draegerwerk/sdc11073/pull/382